extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -10,6 +10,33 @@ class CommunityPageSpecialController extends WikiaSpecialPageController {
 	const MODAL_IMAGE_HEIGHT = 700.0;
 	const MODAL_IMAGE_MIN_RATIO = 0.85;
 	const DEFAULT_MODULES_MAX = 3;
+	// order of permissions is consistent with badges hierarchy in Discussions
+	const PERMISSION_HIERARCHY = [
+		'sysop',
+		'threadmoderator',
+		'content-moderator',
+		'staff',
+		'helper',
+		'vstf'
+	];
+
+//TODO: update when this will be live: https://github.com/Wikia/design-system/pull/69/files
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)